projects/demo/src/modules/components/select/examples/6/index.html

```diff
@@ -21,9 +21,10 @@
         <tui-avatar
             size="xs"
             class="avatar"
-            [avatarUrl]="data.avatarUrl || null"
-            [text]="data.toString()"
-        ></tui-avatar>
+            [src]="data.avatarUrl"
```

Comment:
And such examples are probably meant to be `[src]="data.avatarUrl || (data.toString() | tuiInitials)"`